### PR TITLE
metrics: removed repetitive math

### PR DESCRIPTION
**Pull request by clayjacobs**
Simple calculation that's repeated 10x each instead of once.

### CODE REVIEW

File: metrics/metrics.go [hunk 1]
```diff
@@ -68,18 +68,22 @@ func CollectProcessMetrics(refresh time.Duration) {
...
+		if ReadDiskStats(diskstats[location1]) == nil {
+			diskReads.Mark(diskstats[location1].ReadCount - diskstats[location2].ReadCount)
+			diskReadBytes.Mark(diskstats[location1].ReadBytes - diskstats[location2].ReadBytes)
+			diskWrites.Mark(diskstats[location1].WriteCount - diskstats[location2].WriteCount)
+			diskWriteBytes.Mark(diskstats[location1].WriteBytes - diskstats[location2].WriteBytes)
 		}
 		time.Sleep(refresh)
 	}
-}
+}
```
Comment (karalabe) [inline metrics/metrics.go]: Please fix the file ending.

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 92812006b14ebcd92ba9687b877b3c1f7931d4fd
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by clayjacobs** [comment]
@karalabe closing this pull request and re-requesting to make the changes a single commit

**Comment by karalabe** [comment]
You don't need to close for that. You can amend your commit and force push.
We can also squash merge, so having multiple commits is fine too.

**Pull request closed without merging**